### PR TITLE
Modified signup artist

### PR DESCRIPTION
**Pull request by LauraCourquin**

### CODE REVIEW

**Review by ghost — APPROVED** [review]
ok for me

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

File: backend/src/models/ArtistManager.js [hunk 1]
```diff
@@ -5,9 +5,11 @@ class ArtistManager extends AbstractManager {
 
   insert(artist) {
     return this.connection.query(
-      `insert into ${ArtistManager.table} (user_id,firstname,lastname,phone,address,zipcode,city,nickname,avatar,banner,demo,description,role,cachet) values (?,?,?,?,?,?,?,?,?,?,?,?,?,?)`,
+      `insert into ${ArtistManager.table} (user_id,status,siren,firstname,lastname,phone,address,zipcode,city,nickname,avatar,banner,demo,description,cachet) values (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)`,
       [
         artist.user_id,
+        artist.status,
+        artist.siren,
```
Comment (LoicBrassart) [inline backend/src/models/ArtistManager.js:12]: - 2 field names :heavy_check_mark: 
- 2 variables :heavy_check_mark: 
- 1 question mark ? Has it been tested ? :thinking:

**Review by LoicBrassart — APPROVED** [review]
One doubt, no Evil

File: backend/src/models/ArtistManager.js (hunk 1 above)
Comment (ghost) [reply backend/src/models/ArtistManager.js:12]: Yup it's work

**Pull request merged**